src/Microsoft.DocAsCode.DataContracts.Common/Attributes/UniqueIdentityReferenceAttribute.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.DataContracts.Common.Attributes
+{
+    using System;
+
+    [AttributeUsage(AttributeTargets.Property, AllowMultiple = false, Inherited = true)]
+    public class UniqueIdentityReferenceAttribute : Attribute
```

Comment:
I means should we also define attribute for def and spec?

---
In reply to: [93843278](https://github.com/dotnet/docfx/pull/1042#discussion_r93843278) [](ancestors = 93843278)